contracts/Governance.sol

```diff
@@ -101,8 +101,9 @@ contract Governance is Context, Ownable, ERC165, EIP712 {
     mapping(bytes32 => mapping(address => bool)) public votersRoles;
     mapping(address => mapping(string => bytes32)) public actionsRoles;
     mapping(address => mapping(string => uint256)) public actionsQuorums;
+    mapping(address => bytes32) public protocolsRoles;
```

Comment:
question: is a protocol allowed only a single role, or the `bytes32` used a composition?

Reply:
Single, I think we either stick with Treasury controlled protocols or create a separate role for that as if there is special subDAO for it.